community/ibm-tomcat-dev/RELEASENOTES.md

```diff
@@ -0,0 +1,25 @@
+# Release Notes
+
+## What's new in Chart Version 1.0
```

Comment:
1.0 or 0.1.0?

Reply:
This was a good catch; I'd like this fixed before inclusion as well.